pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -279,8 +279,13 @@ func (asw *actualStateOfWorld) AddVolumeNode(
 			nodesAttachedTo: make(map[types.NodeName]nodeAttachedTo),
 			devicePath:      devicePath,
 		}
-		asw.attachedVolumes[volumeName] = volumeObj
+	} else {
+		// If volume object already exists, it indicates that the information would be out of date.
```

Comment:
Do you think we should glog.Info or glog.V(2).Info, so we can have some indication in the logs?